packages/react/src/Popover/Trigger/PopoverTrigger.test.tsx

```diff
@@ -51,9 +51,29 @@ describe('<Popover.Trigger />', () => {
       expect(trigger).not.to.have.attribute('data-pressed');
     });
 
+    it('should not have the data-popup-open and data-pressed attributes when open by click when `openOnHover=true` and `delay=0`', async () => {
+      const { user } = await render(
+        <Popover.Root delay={0} openOnHover animated={false}>
+          <Popover.Trigger />
+        </Popover.Root>,
+      );
+
+      const trigger = screen.getByRole('button');
+
+      await user.hover(trigger);
+
+      // Closes instantly after opening due to `useClick` prop `stickIfOpen=false`
+      await act(async () => {
+        trigger.click();
+      });
```

Comment:
Perhaps this is why it's configured by default, but it depends on the `delay` you use. If you happen to click <50ms after it opens coincidentally, it's pretty unexpected